README.md

```diff
@@ -31,3 +31,40 @@ With npm:
 With yarn:
 
 > yarn add @nationalbankbelgium/code-style
+
+## Contributing
+
+Please follow our [contribution guidelines](/CONTRIBUTING.md).
+
+To know how to release Stark, refer to [this page](/RELEASE.md).
```

Comment:
Here it should be "code-style" instead of "Stark" :p

Reply:
oops 😜